core/src/main/java/io/seata/core/rpc/processor/client/RmUndoLogProcessor.java

```diff
@@ -48,10 +48,10 @@ public void process(ChannelHandlerContext ctx, RpcMessage rpcMessage) throws Exc
         if (LOGGER.isInfoEnabled()) {
             LOGGER.info("rm handle undo log process:" + msg);
         }
-        handleUndoLogDelete((UndoLogDeleteRequest) msg);
+        doUndoLogDelete((UndoLogDeleteRequest) msg);
     }
 
-    private void handleUndoLogDelete(UndoLogDeleteRequest undoLogDeleteRequest) {
+    private void doUndoLogDelete(UndoLogDeleteRequest undoLogDeleteRequest) {
```

Comment:
Why change handle something to do something? I think this method just handle it, not do it.

Reply:
Because "do" will be better, the meaning of "handle" and "process" is a bit repeated.

Reply:
But the method start of "do", usually means doing real business logic.

Reply:
done.